ServerCore/Pages/Resources/ToolsPartial.cshtml

```diff
@@ -1,49 +1,68 @@
-﻿
-<div class="container">
-    <h1>Puzzle Solving Tools</h1>
+﻿<div class="container">
+    <h1>Puzzle Solving Tools and Techniques</h1>
 
-    <p>If you're wondering how the top teams are able to solve so many puzzles so quickly, part of the answer is that they're probably not relying solely on their brainpower. Puzzles in the Puzzlehunt often involve anagramming, deleting, or otherwise manipulating letters and words to discover new words. This is a task well-suited to computers. Since we can't effectively ban the use of such tools, the least we can do is share the locations of some of them to help level the playing field.</p>
+    <p>If you're wondering how the top teams are able to solve so many puzzles so quickly, part of the answer is that they're probably using computer power to help them. Many puzzle mechanics like anagramming, deleting, or otherwise manipulating letters and words to discover new words are well-suited to computers. Since we can't effectively ban the use of such tools, the least we can do is share the locations of some of them to help level the playing field.</p>
 
-    <h3>Built-in tools</h3>
-    <p>If you're using the browser to look something up, don't forget that there are multiple search engines to try, and all have the possibility of giving different results.  Don't like what you find on Google or Bing?  Try <a href="https://www.seekr.com/" target="_blank">Seekr</a>, <a href="https://www.mojeek.com/" target="_blank">Mojeek</a>, or <a href="https://www.qwant.com/" target="_blank">Qwant</a>.  You also don't have to search with just words; you can use reverse image search to upload an image and look for ones just like it.  Even the browser itself has tools to help you.  For example, Microsoft Edge has built-in screenshotting tools that allow you to draw directly on the screenshot, or web selection tools to only select the relevant parts of a page. Just right-click and look for Web Capture!</p>
+    <br />
+    <h2>General Techniques</h2>
+    <h3>Observation</h3>
+    <p>The first thing you might notice is that some puzzles have no instructions!  In that case, part of the puzzle is figuring out what to do. Sometimes you'll be looking for differences or similarities between things.  Like, maybe a number of pictures all have some theme or root word in common.  Other times, you'll be looking for things that are out of place or arranged suspiciously.  Often, those arrangements are using some sort of <a asp-page="/EventSpecific/Encodings">Encoding</a> to spell out a word or phrase. This is especially true if the puzzle contains more pictures or numbers than words. The title usually also hints at the mechanics.</p>
```

Comment:
Multiple spaces is something that luckily has absolutely no effect on the rendered page since HTML collapses all spaces down to a single space by default.  This is purely my typing style and is way too hard for me to change! lol